tests/test_title_property.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from bokeh.io import save
+from bokeh.plotting import figure
+from bokeh.models import Title
+
+
+def test_figure_title_should_accept_title_or_string():
```

Comment:
This should be two separate tests. Ideally tests just test one thing and have a meaningful test name so it's super easy to pin down what went wrong. (I know this isn't true in a lot of our tests, but we can try).